examples/remotebuildexecution/rbe_system_check/.bazelrc.sample

```diff
@@ -30,9 +30,9 @@ build:remote --auth_enabled=true
 
 # Toolchain/platform flags that work with rbe_autoconfig rule.
 # Note target name must be set to "rbe_default" to use these flags as is.
-build:remote --crosstool_top=@rbe_default//rbe_config_cc:toolchain
-build:remote --host_javabase=@rbe_default//config:jdk
-build:remote --javabase=@rbe_default//config:jdk
+build:remote --crosstool_top=@rbe_default//cc:toolchain
```

Comment:
This is going to impact https://github.com/GoogleCloudPlatform/container-definitions/blob/master/ubuntu1604_rbe/cloudbuild.yaml#L79, just FYI

Reply:
thanks, created https://github.com/GoogleCloudPlatform/container-definitions/issues/20